_posts/2023-04-05-#27255.md

```diff
@@ -10,14 +10,44 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+### Background and prior work
+
+_Miniscript_ is a language for writing (a subset of) Bitcoin Scripts in a structured way, enabling analysis, composition, generic signing and more. _Miniscript_ support in Bitcoin Core has been an ongoing effort, with watch-only support for miniscript descriptors added in `v24.0.1`. For background:
```

Comment:
Maybe note that Miniscript descriptors are only supported under P2WSH in Bitcoin Core and Miniscript itself was anyway only really defined for P2WSH context, not any Bitcoin Script context.

Reply:
I mentioned P2WSH, but I don't want to confuse anyone by trying to explain it further